### ISSUE TITLE
launch failed with as.scm not found

### ISSUE THREAD

**Issue by gpolitis** [shader/arc-openshift#1]
Here's my /var/lib/openshift/.../app-root/logs/launch_errors.log file :

```
default-load-handler: cannot open input file
  path: /var/lib/openshift/.../app-root/data/arc/as.scm
  system error: No such file or directory; errno=2
```

The error comes from [here](https://github.com/shader/arc-openshift/blob/5fdc91186f555441e3bfba69d7b0f6a4a19c0a8b/.openshift/action_hooks/arc.rc#L24). I checked and there's no as.scm file in the [arc repo](https://github.com/arclanguage/anarki.git). Where can I get this file, or maybe this is a typo?

**Comment by shader** [comment]
It seems that akkartik changed as.scm to boot.scm 6 days ago. You can make the change yourself now, and I'll push an update soon.

**Comment by shader** [comment]
Ok, this should be fixed by fa9def0613f9d7ec631852944ec06f9de4e4e34d

**Comment by gpolitis** [comment]
I works, thanks!
